judge/testcase_run.sh

```diff
@@ -212,6 +212,11 @@ runcheck "$RUN_SCRIPT" $RUNARGS \
 	--stderr=program.err --outmeta=program.meta -- \
 	"$PREFIX/$PROGRAM" 2>runguard.err
 
+if [ "$CREATE_WRITABLE_TEMP_DIR" ]; then
+	# Cleanup the TMPDIR as security measure
+	rm -rf "$TMPDIR"
```

Comment:
I think it would be better to `chmod` and `chown` it, so that `$RUNUSER` does not have read or write access to it anymore, but it's still there for auditing/debugging purposes.